src/processors/track-records/handler.ts

```diff
@@ -0,0 +1,45 @@
+import { blockchain, database, EventHandlerInput } from "flair-sdk";
+
+export const processEvent = async (event: EventHandlerInput) => {
+  const provider = await blockchain.getProvider(event.chainId);
+  const transaction = await provider
+    .cached()
+    .getTransactionReceipt(event.txHash);
+
+  if (!event.parsed.name || !event?.parsed?.args) {
+    console.error(
+      `Could not find event name or args for ${event.txHash} # ${event.log.logIndex}`
+    );
+    return;
+  }
+
+  await database.upsert({
+    // Entity type is used to group entities together.
+    // Here we're creating 1 entity per event (Transfer, Borrow, Approval, etc),
+    // based on either entityName defined in the ABI JSON or exact event name.
+    entityType: "Track",
+
+    // Unique ID for this entity.
+    //
+    // Some useful tips:
+    // - chainId makes sure if potentially same tx has happened on different chains it will be stored separately.
+    // - hash and localIndex make sure this event is stored uniquely.
+    // - hash also makes sure with potential reorgs we don't store same event twice.
+    // - logIndex also makes sure if such event has happened multiple times in same tx it will be stored separately.
+    entityId: `${event.parsed.args.id}`,
```

Comment:
```suggestion
    entityId: `${event.chainId}:${event.log.address}:${event.parsed.args.id}`,
    trackId: event.parsed.args.id,
```

Maybe prefix with chainId and contract to ensure uniqueness? You can also keep trackId separately for joins etc.